### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bookercodes**
* Move demo link to the top, so it's easier to find
* I think it would be good to list the features first, to entice people about what's to come 
* Added a note about contributing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,26 +3,12 @@
 [![Star on GitHub][github-star-badge]][github-star]
 [![Tweet][twitter-badge]][twitter]
 
+See the Slack clone in action here: https://pusher.github.io/react-slack-clone
```
Comment (lukejacksonn) [inline README.md]: I usually put these in blockquotes to make them stand out like:

> See the Slack clone in action here: https://pusher.github.io/react-slack-clone

File: README.md [hunk 2]
```diff
@@ -47,9 +35,25 @@ The demo attempts to be feature complete according to documentation [here](https
 - TypingIndicator - to signify to the user that another user is typing in a given room.
 - UserHeader - to display useful information about a given user.
 
+## Running the Slack clone locally
```
Comment (lukejacksonn) [inline README.md]: I'd say Usage is a pretty standard README title. What was the a reasoning behind this title change?

File: README.md [hunk 3]
```diff
@@ -47,9 +35,25 @@ The demo attempts to be feature complete according to documentation [here](https
...
+
+The app depends on a GitHub authentication and user creation endpoint that is hosted [here](https://chatkit-demo-server.herokuapp.com).
+
+
+
 
 [github-star-badge]: https://img.shields.io/github/stars/pusher/react-slack-clone.svg?style=social
 [github-star]: https://github.com/pusher/react-slack-clone/stargazers
-[twitter]: https://twitter.com/intent/tweet?text=Check%20out%20react-testing-library%20by%20%40kentcdodds%20https%3A%2F%2Fgithub.com%2Fkentcdodds%2Freact-testing-library%20%F0%9F%91%8D
+[twitter]: https://twitter.com/intent/tweet?text=Check%20out%20this%20Slack%20clone%20using%20@pusher%20Chatkit%20%F0%9F%91%89https://github.com/pusher/react-slack-clone
```
Comment (lukejacksonn) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (bookercodes) [reply README.md]: I don't mind either way :)

File: README.md (hunk 2 above)
Comment (bookercodes) [reply README.md]: Usage makes sense for a npm module, not so much a demo

Comment (bookercodes) [reply README.md]: I like to reiterate the title sometimes

**Pull request merged**